cross/nettle/Makefile

```
@@ -26,4 +26,11 @@ endif
 ifeq ($(findstring $(ARCH), $(PPC_ARCHES)),$(ARCH))
 CONFIGURE_ARGS = --target=powerpc-unknown-linux-gnu --host=powerpc-unknown-linux-gnu
 endif
-CONFIGURE_ARGS += --enable-shared
+ifeq ($(findstring $(ARCH), $(x86_ARCHES)),$(ARCH))
+CONFIGURE_ARGS = --target=i686-pc-linux-gnu --host=i686-pc-linux-gnu --disable-assembly
+endif
+ifeq ($(findstring $(ARCH), $(x64_ARCHES)),$(ARCH))
+CONFIGURE_ARGS = --target=x86_64-pc-linux-gnu --host=x86_64-pc-linux-gnu
+endif
+CONFIGURE_ARGS += --prefix=$(STAGING_INSTALL_PREFIX) --enable-shared
```

@m4tt075 Sorry but this is not correct. Please let use following `prefix` (`libdir` and `includedir` can be removed too as computed from prefix, well I think so)
```suggestion
CONFIGURE_ARGS += --prefix=$(INSTALL_PREFIX) --enable-shared
```

---

OK. Will try. I've applied the same strategy for the `gnutls` update in another commit. I assume the same fix is needed there. Do you agree?

---

With this change, `evansport` platforms fail with:
```
configure: error: in `/spksrc/spk/tvheadend/work-evansport-6.1/nettle-3.4':
configure: error: cannot find value of GMP_NUMB_BITS
```
That is the same issue I ran into originally. My fix was based on the following information:
http://lists.lysator.liu.se/pipermail/nettle-bugs/2017/006951.html
Any other idea how to fix this?

---

Found the culprit. Unrelated. The configure argument `--disable-assembly` in `nettle` was wrong. It needs to be `--disable-assembler`. Will trigger the next round of test builds and report back. Those won't finish until tomorrow though...

---

@ymartin59 OK, utilizing trial and error with `make arch-evansport-6.1` in `cross/nettle`:
- `--prefix=$(INSTALL_PREFIX)` alone fails with the above error
- `--prefix=$(INSTALL_PREFIX) --with-include-path=$(INSTALL_PREFIX)/include --with-lib-path=$(INSTALL_PREFIX)/lib` fails with the above error
- `--prefix=$(STAGING_INSTALL_PREFIX)` compiles ok, i.e. `lib` and `include` paths apparently ARE computed from the `prefix`
- `--with-include-path=$(STAGING_INSTALL_PREFIX)/include --with-lib-path=$(STAGING_INSTALL_PREFIX)/lib` alone works

I don't know why, but apparently - for x86 only - automatic detection of the `include` and `lib` pathes fails. We don't have to specify a non-standard prefix though. In light of your comment, I'll skip that then and use the same strategy for the `cross/gnutls/Makefile`. Will trigger broader test builds tonight and report back tomorrow.

---

Done. 

P.S.: `--prefix=$(INSTALL_PREFIX)` might(!) be obsolete. At least this is sourced by `spksrc.cross-env.mk` as default setting, and `spksrc.cross-env.mk` is sourced into the `cross/ffmpeg/Makefile` through `spksrc.cross-cc.mk` in return.